pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -3770,6 +3770,21 @@ public boolean isDispatcherPauseOnAckStatePersistentEnabled() {
         return b == null ? false : b.booleanValue();
     }
 
+    @Override
+    public void updateDispatchPauseOnAckStatePersistentEnabled() {
+        super.updateDispatchPauseOnAckStatePersistentEnabled();
+        // Trigger new read if subscriptions has been paused before.
+        if (!brokerService.getPulsar().getConfiguration().isDispatcherPauseOnAckStatePersistentEnabled()) {
+            updateDispatchPauseOnAckStatePersistentEnabled();
```

Comment:
Why do we need to call this method again?

Reply:
Ah, it is a mistake, removed.